### PR TITLE
Eslint autofix for 2.x

### PR DESCRIPTION
**Pull request by bajtos**
@Amir-61 I run `eslint --fix` on 2.x code base to fix at least some of the differences between `master` and `2.x`. Will this make it easier for you to back-port changes from master to 2.x?

I'll leave it up to you to decide whether to land this pull request or close it unmerged.

### CODE REVIEW

**Comment by superkhau** [comment]
LGTM, but amir for final check

File: lib/connectors/memory.js [hunk 1]
```diff
@@ -567,9 +567,9 @@ function applyFilter(filter) {
         return compare(example.neq, value) !== 0;
       }
 
-      if ('between' in example ) {
-        return ( testInEquality({gte:example.between[0]}, value) &&
-                 testInEquality({lte:example.between[1]}, value) );
+      if ('between' in example) {
+        return (testInEquality({ gte:example.between[0] }, value) &&
+                 testInEquality({ lte:example.between[1] }, value));
```
Comment (Amir-61) [inline lib/connectors/memory.js]: Just out of curiosity if we break a line to multiple lines, what is the preferred style?

Based on `Eslint --fix` it seems it adds 9 spaces to indent the second line; should it be two spaces?
```
return (testInEquality({ gte:example.between[0] }, value) &&
  testInEquality({ lte:example.between[1] }, value));
```

Just wanted to know the preference in the team; also the proceeding lines do not have any more indentation, I believe, right?

Example:
```
return (testInEquality({ gte:example.between[0] }, value) &&
  testInEquality({ lte:example.between[1] }, value) &&
  testInEquality({ lte:example.between[2] }, value));
```

**Comment by Amir-61** [comment]
@slnode test please

File: lib/connectors/memory.js (hunk 1 above)
Comment (superkhau) [inline lib/connectors/memory.js]: @bajtos can field this one. Please update the style-guide too with this answer as I'm also curious.

**Comment by Amir-61** [comment]
LGTM

File: lib/connectors/memory.js (hunk 1 above)
Comment (bajtos) [inline lib/connectors/memory.js]: Good catch! Apparently eslint doesn't care whether its 2 spaces, 8 spaces or 9 spaces in this case.

Your example looks exactly as what I would write myself, @Amir-61:
>```js
return (testInEquality({ gte:example.between[0] }, value) &&
  testInEquality({ lte:example.between[1] }, value) &&
  testInEquality({ lte:example.between[2] }, value));
```

**Comment by bajtos** [comment]
I have rebased on top of the current 2.x. While I was addressing Amir's comment, I fixed all eslint issues I found in the memory connector (see the second commit).

File: lib/connectors/memory.js (hunk 1 above)
Comment (bajtos) [inline lib/connectors/memory.js]: > Please update the style-guide too with this answer as I'm also curious.

https://github.com/strongloop-internal/loopback-knowledge-base/pull/10

**Comment by bajtos** [comment]
@Amir-61 PTAL one more time

**Comment by Amir-61** [comment]
LGTM

**Pull request merged**